sql/log_event.cc

```diff
@@ -2170,7 +2170,7 @@ log_event_print_value(IO_CACHE *file, const uchar *ptr,
       struct timeval tm;
       my_timestamp_from_binary(&tm, ptr, meta);
       int buflen= my_timeval_to_str(&tm, buf, meta);
-      my_b_write(file, buf, buflen);
+      my_b_write(file, reinterpret_cast<uchar*>(buf), buflen);
```

Comment:
Likewise

Reply:
No cast is needed, `buf` is already `uchar*` here.

Reply:
Fixed